client/core/core.go

```diff
@@ -2043,7 +2043,7 @@ func (c *Core) handleReconnect(uri string) {
 // NOTE: This handler is only used to post notifications to the user.
 func (c *Core) handleConnectEvent(uri string, connected bool) {
 	c.connMtx.Lock()
-	if dc, found := c.conns[uri]; found {
+	if dc, found := c.conns["https://"+uri]; found {
```

Comment:
Did something change here? could be something other that https?

Reply:
It's eventually not going to have a scheme at all, but for the map lookup, definitely should not prepend with this.